src/types/core.ts

```diff
@@ -116,3 +120,11 @@ export interface ProviderDetails {
   url?: string;
   apiKey?: string;
 }
+
+export interface OcspResponse {
```

Comment:
what do you think to add a comment to the spec about ocsp response format ?

Reply:
yes will need to update that (you mean the adr right?)

Reply:
I didn't mean the adr but I'm fine with a link to the adr too